src/testData/mockData.json

```diff
@@ -0,0 +1,101 @@
+{
+    "DEFAULT_URL": "https://unstoppabledomains.com/api/v1",
+    "API_RESPONSE": {
```

Comment:
I would group request/response for a single mock together instead of all responses and requests together like:

``` js
{
  zilliqaRegistryState: {
    REQUEST: ...,
    RESPONSE: ...,
  }
 etc
}